msteams-platform/concepts/design/enable-app-customization.md

```diff
@@ -29,27 +32,25 @@ You can't test this feature during development. App customization isn't supporte
 
 Provide guidelines for customers (specifically Teams admins) who want to customize your app. For more information, see [customize apps in Teams](/MicrosoftTeams/customize-apps).
 
-## Hide Teams app until admin approves
+## Hide Teams app from users
```

Comment:
This is a completely different use case. Why is it in this article? Can we find a related article for this content and remove from this article?

CC @surbhigupta12

Reply:
@v-ajiths , for now, let's update the sub-heading to `Block apps by default for users until an admin approves`.

We shall split it later. CC @surbhigupta12 , let's have an ADO to track this.

Reply:
updated as suggested